wallet/how-to/use-non-evm-networks/starknet/about-get-starknet.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Learn about how `get-starknet` interacts with MetaMask.
+sidebar_position: 8
+---
+
+# About `get-starknet`
+
+[`get-starknet`](https://github.com/starknet-io/get-starknet) is a library that simplifies Starknet
+network interactions.
+It works with the [Starknet Snap](https://snaps.metamask.io/snap/npm/consensys/starknet-snap/) to
+enable dapps to interact with users' Starknet accounts in MetaMask.
+
+When you integrate `get-starknet` into your dapp, it creates a `WalletAccount` object, which acts as
+the connection between the dapp and MetaMask, and manages Starknet interactions.
+This allows users to send Starknet transactions, sign Starknet messages, and manage Starknet
+accounts within MetaMask, and this functionality can be extended to multiple wallets in the Starknet
+ecosystem.
+
+## How `get-starknet` and MetaMask interact
+
+A dapp with `get-starknet` installed interacts with MetaMask as follows:
+
+1. The dapp uses `get-starknet` to request the user connect to MetaMask.
+   MetaMask automatically requests the user to add the Starknet Snap, if it's not already present.
+
+1. After the dapp is connected to MetaMask and the Starknet Snap, `get-starknet` receives a Starknet
```

Comment:
@stanleyyconsensys Sorry that's not a typo. The markdown will increment. If you check the Vercel build it will show sequential numbering.  I just chose to do it like that in case we add or remove another step - that way we will not have to renumber everything ;)